src/FlyWithLua.cpp

```diff
@@ -6621,8 +6617,7 @@ bool ReadAllScriptFiles()
     // run through the init script
     std::ostringstream oss_IntPathAndName;
     oss_IntPathAndName << internalsDir << "FlyWithLua.ini";
-    std::string IntPathAndName;
-    IntPathAndName = oss_IntPathAndName.str();
+    auto IntPathAndName = oss_IntPathAndName.str();
```

Comment:
Same here. I don't think `IntPathAndName` is used anywhere but in logMsg(), so no need to create an intermediate variable.

Reply:
I looks like the ReadScriptFile is using it.

logMsg(logToAll, "FlyWithLua Info: FlyWithLua.ini full path ");
    logMsg(logToAll, IntPathAndName.c_str());
    if (ReadScriptFile(IntPathAndName.c_str()))

Reply:
Ah, ok, I missed that. Although conversion to a C string is unnecessary (`c_str()`).